lib/asciidoctor/js/opal_ext/file.rb

```diff
@@ -76,25 +76,87 @@ def readlines
     File.readlines(@path)
   end
 
-  def self.readlines(path, separator = $/)
-    content = File.read(path)
-    content.split(separator)
-  end
+  class << self
+
+    def read(path)
+      case JAVASCRIPT_IO_MODULE
+      when 'node'
```

Comment:
We should remove the 'node' case here as this would reintroduce what we removed in master.

Reply:
As make sure the case in the constructor doesn't get reintroduced.

Reply:
Bad rebase on this file, sorry 😞